scenarios/maritime/batch/batchSimulation.m

```diff
@@ -0,0 +1,88 @@
+function batchSimulation(simulationSeed, sweepEnabled)
```

Comment:
That does sweepEnabled do? Can this be added to the function fingerprint?

Reply:
Sure, good suggestion. It is a boolean flag to tell the simulation instance whether to run with the antenna sweeping algorithm enabled or not to generate comparative scenarios